index.html

```diff
@@ -5,12 +5,73 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+    <title>Soraiya's Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cardo&family=Karla:wght@300&family=Lora&family=Vollkorn&family=Vollkorn+SC&display=swap"
+        rel="stylesheet">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <section>
+    <header>
+        <div><img class="logo" src="/img/logo.jpeg"></div>
```

Comment:
Your logo isn't showing. Add a dot before the first slash: `./img/logo.jpeg`
This means "the `img` folder is at the same level as this current file (index.html)"